src/asdf_standard/_versioning.py

```diff
@@ -0,0 +1,42 @@
+import os
+import warnings
+
+from .exceptions import DevCoreSchemasWarning
+
+DEV_ENV_VAR = "ASDF_DEV_CORE_SCHEMAS"
+
+stable_core_schema_versions = (
+    "1.0.0",
+    "1.1.0",
+    "1.2.0",
+    "1.3.0",
+    "1.4.0",
+    "1.5.0",
+    "1.6.0",
+)
+
+dev_core_schema_versions = ("1.7.0",)
```

Comment:
Is there a check to validate that the `dev` schemas are in the `dev` subdirectory? Otherwise I foresee a situation where this line isn't kept up to date, or vice versa

Reply:
Great idea! Let me add that check.